### PR TITLE
Ng 4

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: templates/web-angular/pubspec.yaml [hunk 1]
```diff
@@ -8,20 +8,20 @@ environment:
   sdk: '>=1.23.0 <2.0.0'
```
Comment (chalin) [inline templates/web-angular/pubspec.yaml]: Probably want 1.24 as lower bound.

**Comment by kevmoo** [comment]
Yup!

**Comment by chalin** [comment]
The rest LGTM (though you might need to regenerate).

**Comment by kevmoo** [comment]
Oops...yup

**Comment by kevmoo** [comment]
@kwalrath @matanlurey – I raw w/ the LGTM from @chalin 

I'd like to get this out ASAP.

**Comment by kwalrath** [comment]
@matanlurey: Will angular 4.0.0 work with 1.24? I'm asking because I'm not sure whether we provide kiosks at developer events, but if we do, they probably still have 1.23 (like they did for google i/o).

**Pull request merged**